### DISCUSSION TITLE
[Feature Request] Disable rules (or completely turn off rome linter) for a file

### DISCUSSION THREAD

**Discussion by llc1123** [rome/tools#4526] [Suggestions]
### Description

I often use something like `/* eslint-disable @typescript-eslint/no-explicit-any */` in some files, and I cannot find a replacement in rome linter.
#3792

**Comment by ematipico** [comment]
That's definitely something that we want to implement!

**Comment by ysageev** [comment]
Just want to bump this.  `eslint-disable` is completely ignored when placed at the top of the file.
